packages/reports/src/Components/Errors/error-component.jsx

```diff
@@ -14,28 +14,30 @@ const ErrorComponent = ({
 }) => {
     const refresh_message = should_show_refresh ? localize('Please refresh this page to continue.') : '';
 
-    return is_dialog ? (
-        <Dialog
-            title={header || localize('There was an error')}
-            is_visible
-            confirm_button_text={redirect_label || localize('Ok')}
-            onConfirm={redirectOnClick || (() => location.reload())}
-        >
-            {message || localize('Sorry, an error occured while processing your request.')}
-        </Dialog>
-    ) : (
-        <PageError
-            header={header || localize('Something’s not right')}
-            messages={
-                message
-                    ? [message, refresh_message]
-                    : [localize('Sorry, an error occured while processing your request.'), refresh_message]
-            }
-            redirect_urls={[routes.trade]}
-            redirect_labels={[redirect_label || localize('Refresh')]}
-            buttonOnClick={redirectOnClick || (() => location.reload())}
-        />
-    );
+    if (is_dialog) {
+        return (
+            <Dialog
+                title={header || localize('There was an error')}
+                is_visible
+                confirm_button_text={redirect_label || localize('Ok')}
+                onConfirm={redirectOnClick || (() => location.reload())}
+            >
+                {message || localize('Sorry, an error occured while processing your request.')}
+            </Dialog>
+        );
+    } else if (header && message) {
+        return (
+            <PageError
+                header={header}
+                messages={[message, refresh_message]}
+                redirect_urls={[routes.trade]}
+                redirect_labels={[redirect_label || localize('Refresh')]}
+                buttonOnClick={redirectOnClick || (() => location.reload())}
+            />
+        );
```

Comment:
```suggestion
    } else if (header && message) {
        return (
            <PageError
                header={header}
                messages={[message, refresh_message]}
                redirect_urls={[routes.trade]}
                redirect_labels={[redirect_label || localize('Refresh')]}
                buttonOnClick={redirectOnClick || (() => location.reload())}
            />
        );
```

See comments marked with 🐔 . This i think can be abstracted to its own component. `PageRefreshError` or something.